cost-analyzer/values.yaml

```diff
@@ -40,6 +40,7 @@ global:
     # alertConfigs:
       # frontendUrl: http://localhost:9090 # optional, used for linkbacks
       # globalSlackWebhookUrl: https://hooks.slack.com/services/T00000000/B00000000/XXXXXXXXXXXXXXXXXXXXXXXX # optional, used for Slack alerts
+      # globalTeamsWebhookUrl: https://xxxxx.webhook.office.com/webhookb2/XXXXXXXXXXXXXXXXXXXXXXXX/IncomingWebhook/XXXXXXXXXXXXXXXXXXXXXXXX # optional, used for Microsoft Teams alerts
```

Comment:
@dwbrown2 thoughts on the naming for this? `globalMSTeamsWebhookUrl`?

Reply:
@nikovacevic Is this PR blocking getting the frontend team unblocked?  Will reach out to Webb to get eyes on it but if if it is low risk would just roll if we dont here back.

Reply:
I think the name makes sense. Its consistent with the naming for slack - 'globalSlackWebhookUrl'

Reply:
@nickcurie happy to approve after you've made that change!